Tools/clinic/clinic.py

```diff
@@ -1016,6 +1017,9 @@ def parser_body(
                 if has_optional:
                     parser_code.append("skip_optional:")
             else:
+                for parameter in parameters:
+                    parameter.converter.use_converter()
```

Comment:
Should not this belong to the `if fastcall:` scope in L1025?

Reply:
The 3 code path below use `{parse_arguments}` and so use converters, no? I can be wrong, I'm not sure.

Reply:
I don't know. I think the added `use_converter` method and the refactoring makes it hard to understand what's happening.

Reply:
The last path uses `PyArg_ParseTuple`, so no converters are used, right?